build-parent/pom.xml

```diff
@@ -523,9 +548,7 @@
             <id>format</id>
             <activation>
                 <activeByDefault>true</activeByDefault>
-                <property>
-                    <name>!no-format</name>
-                </property>
```

Comment:
I don't understand this change.

Reply:
neither do it ;/

wrong profile it was done to - pushed a fix.